packages/nimble-components/src/select/tests/select.spec.ts

```diff
@@ -411,5 +414,32 @@ describe('Select', () => {
             await pageObject.clickSelect();
             expect(pageObject.isNoResultsLabelVisible()).toBeFalse();
         });
+
+        it('clicking disabled option does not cause filtered options to reappear', async () => {
```

Comment:
How oddly specific this test is makes me laugh since I saw the behavior before you fixed this bug 😄 

However, I'm wondering if it should be something broader like "clicking disabled option does not cause the select to change state". Then, it could check:
- the options stay the same
- the entered filter text stays the same
- the select stays open